README.md

```diff
@@ -22,7 +22,7 @@ on:
 jobs:
   call-setup-env-workflow:
     # Docs at: https://github.com/ASFOpenSARlab/osl-utils?tab=readme-ov-file#reusable-setup-envyaml
-    uses: ASFOpenSARlab/osl-utils/.github/workflows/reusable-setup-env.yaml@v0.0.2
+    uses: ASFOpenSARlab/osl-utils/.github/workflows/reusable-setup-env.yaml@v<LATEST_TAG>
```

Comment:
`<LATEST_TAG>` sounds like `:latest`... 🤔 Not sure if that'll be confusing.

Reply:
I agree. I tried to leave the `v...` to help with that. If I did `v*` though, they might copy-paste it without changing the var. Any idea what would be better? `v<LATEST_RELEASE>` maybe?